docs/docs/release-download.md

```diff
@@ -1,7 +1,8 @@
 
 
-These are built BigDL packages including dependency and python files. You can download these packages instead of building them by yourself. This is useful when you want to do something like run some examples or develop python code.
+These are built BigDL packages including dependency and Python files. You can download these packages instead of building them by yourself. This is useful when you want to do something like run some examples or develop Python code.
 
+NOTE: __Python 2.7__ and __Python 3.5__ are recommended.
```

Comment:
Only python 2.7 and 3.5 are currently supported.

Reply:
OK~